data-structures/linked_list/mergeList.py

```diff
@@ -0,0 +1,33 @@
+# from linked_list import LinkedList
+
+
+def ll_merge(arg2, arg1):
```

Comment:
You got the names backwards. The file is supposed to be named `ll_merge` and the function is supposed to be named `mergeList`.